.github/workflows/build-container.yml

```diff
@@ -11,7 +11,7 @@ on:
       - 'docker/docker-entrypoint.sh'
       - 'workflows/build-container.yml'
     tags:
-      - 'v*'
+      - 'v*.*.*'
```

Comment:
With the new release workflow triggering on tags, and use making two tags per release (e.g. `v3.5.0` and `v3-latest`), this was running twice. The narrower pattern ensures we only build the container once on release tags.

Reply:
Do we actually still need `v3-latest`? I introduced it during a period when we were still actively bug-fixing the 2.0 branch and we had reasons to tag the latest release on each branch, but that time is long past.

Reply:
I think it's safe to remove

Reply:
Removed